### PR TITLE
chore: use `Deno.Command`, tweak provider timeouts

### PR DESCRIPTION
**Pull request by kratico**
Resolves https://github.com/paritytech/capi/issues/650

Migrate to Deno.Command from Deno.run

This is the same as #664 but with longer deadline to execute `polkadot` or `zombienet`

### CODE REVIEW

File: main.ts [hunk 1]
```diff
@@ -70,10 +70,12 @@ if (!running) {
 
 async function onReady() {
   if (cmd.length) {
-    const process = Deno.run({ cmd })
-    const status = await process.status()
+    const [commandPath, ...args] = cmd
+    const command = new Deno.Command(commandPath!, { args })
+    const process = command.spawn()
+    process.unref()
```
Comment (tjjfvi) [inline main.ts:76]: I don't think this is right... we explicitly want to keep this process open until the inner one closes

Comment (kratico) [reply main.ts]: removed

**Comment by harrysolovay** [comment]
@kratico this appears to be similar to #675? Wanna close this one?

**Comment by kratico** [comment]
> @kratico this appears to be similar to #675? Wanna close this one?

@harrysolovay  closed

**Review by harrysolovay — APPROVED** [review]

**Review by tjjfvi — APPROVED** [review]

**Pull request merged**